css/css-break/grid/monolithic-overflow-010-print-ref.html

```diff
@@ -0,0 +1,19 @@
+<div style="height:40vh">
+  Test passes if there is two purple boxes each on both page 1 and page 2 in print mode. (Ctrl+P)
```

Comment:
A few nits:
(1) the text here doesn't quite match the text in the testcase (`each` is here but missing in the testcase)
(2) Worth including `<!DOCTYPE html>` here in the reference case; otherwise it's rendered in quirks mode, which in this case doesn't impact the rendering, but it's a good best-practice (particularly to avoid comparing a standards-mode testcase against a quirks-mode reference case)
(3) If I take the human-readable instructions at face value: really, most viewers probably wouldn't see **any purple boxes** at all on page 1 or page 2, because browsers disable printing `background` by default.  We enable backgrounds for test runs, but not for human viewers (so that e.g. a page with `body{background:yellow}` doesn't use up all your yellow ink).  Maybe this isn't a big deal, but if you want to avoid confusing human viewers, you might expand (Ctrl+P) to (Ctrl+P, with "print backgrounds" enabled) or somesuch?

Reply:
Good points, I didn't realize I forgot to edit the reference after I edited the test case.
As for the "print backgrounds" part, I'll also add it to the other (very similar) test I made earlier.